src/xenia/kernel/xam/xam_ui.cc

```diff
@@ -115,9 +111,9 @@ dword_result_t XamShowMessageBoxUI(dword_t user_index, lpstring_t title_ptr,
   XELOGD(
       "XamShowMessageBoxUI(%d, %.8X(%S), %.8X(%S), %d, %.8X(%S), %d, %X, %.8X, "
       "%.8X)",
-      user_index, title_ptr, title.c_str(), text_ptr, text.c_str(),
-      button_count, button_ptrs, all_buttons.c_str(), active_button, flags,
-      result_ptr, overlapped_ptr);
+      user_index, title, title.value(), text, text.value(), button_count,
```

Comment:
`title.value()` should be `!title ? L"" : title.value().c_str()`, and `text.value()` should be `text.value().c_str()`.